layers/+lang/javascript/packages.el

```diff
@@ -30,9 +30,7 @@
                            :repo "emacs-lsp/lsp-javascript"))
         skewer-mode
         tern
-        web-beautify
-        yasnippet
```

Comment:
Doesen't this remove snippet functionality from the `javascript` layer? (for example when developing for nodejs without typescript/react).

Reply:
It shouldn't, no. Snippets are enabled via the `auto-completion` layer. All of the stuff that this does only has to do with enabling js-mode's snippets in other, non-js-mode-derived modes. yasnippet enables js-mode snippets in js-mode derived modes automatically.